ingestion-beam/pom.xml

```diff
@@ -138,6 +138,12 @@
             <artifactId>slf4j-jdk14</artifactId>
             <version>1.7.30</version>
         </dependency>
+        <dependency>
+            <groupId>com.mozilla.telemetry</groupId>
+            <artifactId>ingestion-core</artifactId>
+            <version>0.1-SNAPSHOT</version>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
I had to add this to make it compile in IntelliJ. I can also git ignore this if it should be removed.

Reply:
Okay, needs to be removed otherwise CircleCI cannot compile